packages/react/src/components/Modal/Modal.js

```diff
@@ -167,30 +173,18 @@ export default class Modal extends Component {
     modalHeading: '',
     modalLabel: '',
     selectorPrimaryFocus: '[data-modal-primary-focus]',
-    focusTrap: true,
     hasScrollingContent: false,
   };
 
   button = React.createRef();
   outerModal = React.createRef();
   innerModal = React.createRef();
+  startSentinel = React.createRef();
```

Comment:
Is there a clearer plain-language way we can name this as opposed to "sentinel". It's difficult jargon to even Google a simple definition of. I assume you're meaning like a [sentinel node](https://en.wikipedia.org/wiki/Sentinel_node), but there's probably a friendlier way to describe that.

Reply:
@dakahn Hope this link shed some light: https://developers.google.com/web/fundamentals/accessibility/focus/using-tabindex

Reply:
My question wasn't "why did you name this like this" -- I get why. My question was if we could name this in plainer and simpler to understand language?

Reply:
Maybe "start trap" and "end trap"?